packages/core/src/with-router.tsx

```diff
@@ -4,6 +4,8 @@ import {WithRouterProps as WithNextRouterProps} from "next/dist/client/with-rout
 import {useParams, extractRouterParams} from "./use-params"
 import {useRouterQuery} from "./use-router-query"
 
+export {RouterContext} from "next/dist/next-server/lib/router-context"
```

Comment:
is this something we should write a test for in case this file moves?

Reply:
The build will fail if it moves

Reply:
🤦‍♂️ it's been a long day